kalite/topic_tools/annotate.py

```diff
@@ -50,97 +50,71 @@ def update_content_availability(content_list, language="en"):
                     else:
                         subtitle_langs[filename] = [lc]
 
-    # English-language exercises live in application space, translations in user space
```

Comment:
Everything lives in user space now, right? If the content databases don't exist, they'll be created in USER_DATA_ROOT somewhere?